scoresheets/HandMeThat.tex

```diff
@@ -0,0 +1,18 @@
+
+The maximum time for this test is 15 minutes.
```

Comment:
I think previously stage 2 tasks were about 10 minutes. With how the schedule works all stage 2 tasks need to be of equal length. Not saying we need to change this here but it would mean all stage 2 tasks need to also be 15 minutes long.